stream.js

```diff
@@ -27,7 +31,14 @@ function WebSocketStream(target, protocols, options) {
     socket = target
   // otherwise make a new one
   } else {
-    socket = new WS(target, protocols, options)
+    // special constructor treatment for native websockets in browsers, see
+    // https://github.com/maxogden/websocket-stream/issues/82
+    if (isNative && isBrowser && !protocols) {
+      socket = new WS(target)
```

Comment:
Yes, but I'm talking about the optional **protocols** argument which is in the spec.